dockertest/output.py

```diff
@@ -723,7 +723,7 @@ def mustpass(cmdresult, failmsg=None):
     return cmdresult
 
 
-def mustfail(cmdresult, failmsg=None):
+def mustfail(cmdresult, expected_status, failmsg=None):
     """
```

Comment:
Might do a quick full-repo grep (or cscope) to see if there are any ``mustfail()`` callers that pass two positional-parameters, I suspect there may be either none, or a billion.  I was planning on merging this PR into the new 0.8.5 release, so the API changes are segregated.  Otherwise, just FYI -  consider when adding parameters, usually sticking them at the end as a keyword parameter is safest, and have it default to previous behaviour.